benchmark/api/query.benchmark.cpp

```diff
@@ -32,7 +33,8 @@ class QueryBenchmark {
     HeadlessFrontend frontend { { 1000, 1000 }, 1 };
     Map map { frontend, MapObserver::nullObserver(),
               MapOptions().withMapMode(MapMode::Static).withSize(frontend.getSize()),
-              ResourceOptions().withCachePath("benchmark/fixtures/api/cache.db").withAssetPath(".").withApiKey("foobar") };
+              ResourceOptions().withCachePath("benchmark/fixtures/api/cache.db").withAssetPath(".").withApiKey("foobar"),
+              ClientOptions() };
```

Comment:
I guess `ClientOptions()` means that you use some default object. Would it also be possible to make the client option argument optional?

Reply:
Did that now for outer facing interfaces (`Map` and `MapSnapshotter`).